srcpkgs/highway/template

```diff
@@ -1,23 +1,33 @@
 # Template file for 'highway'
 pkgname=highway
-version=1.1.0
+version=1.2.0
 revision=1
 build_style=cmake
-configure_args="-DHWY_SYSTEM_GTEST=ON -DHWY_ENABLE_EXAMPLES=OFF"
+configure_args="-DHWY_SYSTEM_GTEST=ON -DHWY_ENABLE_EXAMPLES=OFF -DBUILD_SHARED_LIBS=ON"
 checkdepends="gtest-devel"
 short_desc="C++ library providing portable SIMD/vector intrinsics"
 maintainer="Joshua Krämer <joshua@kraemer.link>"
 license="Apache-2.0"
 homepage="https://github.com/google/highway"
 changelog="https://raw.githubusercontent.com/google/highway/master/debian/changelog"
 distfiles="https://github.com/google/highway/archive/${version}.tar.gz"
-checksum=354a8b4539b588e70b98ec70844273e3f2741302c4c377bcc4e81b3d1866f7c9
+checksum=7e0be78b8318e8bdbf6fa545d2ecb4c90f947df03f7aadc42c1967f019e63343
 
 if [ -z "$XBPS_CHECK_PKGS" ]; then
 	configure_args+=" -DBUILD_TESTING=OFF"
 fi
 
-# Workaround for i686, requires GCC 13+
-# See: https://github.com/google/highway/issues/1488
-CFLAGS="-fexcess-precision=standard"
-CXXFLAGS="-fexcess-precision=standard"
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*) configure_args+=" -DHWY_CMAKE_SSE2=ON";;
```

Comment:
Don't enable SSE2 on i686.

Reply:
I asked about sse2 on i686 in `#xbps`

there are several other packages which enable sse2 on i686, including it being the default for zig packages

Reply:
min support is documented as p4/sse2 https://docs.voidlinux.org/installation/index.html#base-system-requirements

Reply:
We do patch packages to disable SSE2 and this was disabled in a package after being enabled as recently as a few days ago: https://github.com/void-linux/void-packages/commit/b5d8482696002b1bda82ad9b57a52a8873325d3a

Reply:
yeah i know it won't do anything on x86_64. I'll remove it there